src/Tutor.Infrastructure/Database/TutorContext.cs

```diff
@@ -1,30 +1,36 @@
 ﻿using Microsoft.EntityFrameworkCore;
 using Tutor.Core.ContentModel.LearningObjects;
-using Tutor.Core.ContentModel.LearningObjects.ArrangeTasks;
-using Tutor.Core.ContentModel.LearningObjects.Challenges;
-using Tutor.Core.ContentModel.LearningObjects.Challenges.FulfillmentStrategy;
-using Tutor.Core.ContentModel.LearningObjects.Challenges.FulfillmentStrategy.MetricChecker;
-using Tutor.Core.ContentModel.LearningObjects.Challenges.FulfillmentStrategy.NameChecker;
-using Tutor.Core.ContentModel.LearningObjects.Questions;
 using Tutor.Core.ContentModel.Lectures;
-using Tutor.Core.KnowledgeComponentModel.KnowledgeComponents;
+using Tutor.Core.DomainModel.AssessmentEvents.ArrangeTasks;
+using Tutor.Core.DomainModel.AssessmentEvents.Challenges;
+using Tutor.Core.DomainModel.AssessmentEvents.Challenges.FulfillmentStrategy;
+using Tutor.Core.DomainModel.AssessmentEvents.Challenges.FulfillmentStrategy.MetricChecker;
+using Tutor.Core.DomainModel.AssessmentEvents.Challenges.FulfillmentStrategy.NameChecker;
+using Tutor.Core.DomainModel.AssessmentEvents.Questions;
+using Tutor.Core.DomainModel.InstructionalEvents;
+using Tutor.Core.DomainModel.Units;
 using Tutor.Core.LearnerModel.Learners;
 using Tutor.Core.ProgressModel.Feedback;
 using Tutor.Core.ProgressModel.Progress;
 using Tutor.Core.ProgressModel.Submissions;
+using KnowledgeComponent = Tutor.Core.DomainModel.KnowledgeComponents.KnowledgeComponent;
```

Comment:
Once we remove the content model we should remember to fix these using to follow the current convention.